keyboards/handwired/108keyTrackpoint/keymaps/dvorak/keymap.c

```diff
@@ -0,0 +1,14 @@
+#include "108keyTrackpoint.h"
```

Comment:
This should be:

```suggestion
#include QMK_KEYBOARD_H
```

which will include the needed files automatically.